site/content/en/docs/Reference/API/Kustomization File/nameSuffix.md

```diff
@@ -4,7 +4,18 @@ linkTitle: "nameSuffix"
 type: docs
 weight: 13
 description: >
-    Appends the value to the names of all resources and references.
+    Add suffixes to the names of all resources.
 ---
+`apiVersion: kustomize.config.k8s.io/v1beta1`
 
-The Tasks section contains examples of how to use [`nameSuffix`](/docs/tasks/namespaces_and_names/).
+See the [Tasks section] for examples of how to use `nameSuffix`.
+
+### nameSuffix
+Add suffixes to the names of all resources.
+
+* **nameSuffix** (string)
+
+    NameSuffix will add a suffix the names of all resources mentioned in the Kustomization file including generated resources such as ConfigMaps and Secrets.
```

Comment:
```suggestion
    NameSuffix will add a suffix the names of all resources mentioned in the Kustomization file including generated resources such as ConfigMaps and Secrets.
```

nit. suffix `to` the names

Reply:
Updated per your suggestion. Resolving comment.